docs/developer/README.md

```diff
@@ -0,0 +1,206 @@
+# Requirements
```

Comment:
So we will now have a `developer` and `development` folder under docs, each with a top level README... what's the difference intended to be between the two?  Should we combine them and carve out separate content under one directory maybe?